code/game/machinery/cryopod.dm

```diff
@@ -0,0 +1,433 @@
+/*
+ * Cryogenic refrigeration unit. Basically a despawner.
+ * Stealing a lot of concepts/code from sleepers due to massive laziness.
+ * The despawn tick will only fire if it's been more than time_till_despawned ticks
+ * since time_entered, which is world.time when the occupant moves in.
+ * ~ Zuhayr
+ */
+
+
+//Main cryopod console.
+
+/obj/machinery/computer/cryopod
+	name = "cryogenic oversight console"
+	desc = "An interface between crew and the cryogenic storage oversight systems."
+	icon = 'icons/obj/Cryogenic2.dmi'
+	icon_state = "cellconsole_1"
+	circuit = /obj/item/circuitboard/cryopodcontrol
+	density = FALSE
+	interact_offline = TRUE
+	req_one_access = list(ACCESS_HEADS, ACCESS_ARMORY) //Heads of staff or the warden can go here to claim recover items from their department that people went were cryodormed with.
+	var/mode = null
+
+	//Used for logging people entering cryosleep and important items they are carrying.
+	var/list/frozen_crew = list()
+	var/list/frozen_items = list()
+
+	var/storage_type = "crewmembers"
+	var/storage_name = "Cryogenic Oversight Control"
+	var/allow_items = TRUE
+
+/obj/machinery/computer/cryopod/attack_ai()
+	attack_hand()
+
+/obj/machinery/computer/cryopod/attack_hand(mob/user = usr)
+	if(stat & (NOPOWER|BROKEN))
+		return
+
+	user.set_machine(src)
+	add_fingerprint(usr)
```

Comment:
You're assigning `usr` to `user`, might as well remain consistent using the variable `user`?